src/graphql/resolvers/index.js

```diff
@@ -3,12 +3,10 @@ import { resolvers as graphScalars } from 'graphql-scalars';
 import customScalars from './scalars/date';
 import queries, { fieldResolvers as qFieldResolvers } from './queries';
 import mutations, { fieldResolvers as mFieldResolvers } from './mutations';
-// import federation from './federation';
```

Comment:
ya know git works pretty good for tracking history...no need to leave commented out stuff in the code ;)

Reply:
Lol. Missed that one.